packages/protocol/test/L2/V1TaikoL2.test.ts

```diff
@@ -14,19 +16,17 @@ describe("V1TaikoL2", function () {
             await ethers.getContractFactory("LibTxDecoder")
         ).deploy()
 
-        const v1TaikoL2Factory = await ethers.getContractFactory("V1TaikoL2", {
-            libraries: {
-                LibTxDecoder: libTxDecoder.address,
-            },
-        })
-        const v1TaikoL2 = await v1TaikoL2Factory.deploy(addressManager.address)
-
-        return { v1TaikoL2 }
-    }
+        v1TaikoL2 = await (
+            await ethers.getContractFactory("V1TaikoL2", {
```

Comment:
ditto, and im not sure ` ethers.getContractFactory("V1TaikoL2", ...)` works now? I think this contract factory can not be found now since the name change?

Reply:
Had to clone repo again, working on it!